helm/chart/router/Chart.yaml

```diff
@@ -25,4 +25,4 @@ version: 1.0.0-rc.5
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v1.2.0"
+appVersion: "v1.2.1"
```

Comment:
We need to rev the chart version as well: rc-6 seems appropriate.

I think the next release could be 1, since we have now had an rc rev which contained no changes apart from the appversion...

Reply:
Doh.  Good catch.  Fixing!

Reply:
Fixed in fb06ac84.  Please take another look!